cas-server-support-saml-googleapps/src/main/java/org/jasig/cas/support/saml/authentication/principal/GoogleAccountsServiceFactory.java

```diff
@@ -63,7 +63,7 @@ public GoogleAccountsServiceFactory() {}
 
     /**
      * Init public and private keys.
-     * @throws Exception if key creation encountered an error.
+     * @throws RuntimeException if key creation encountered an error.
```

Comment:
Sounds good. If you are using IDEA, you may want to tone down the inspections. Our de facto grump! is Checkstyle (and Findbugs to some degree). If they are happy, we are happy :) Unchecked exceptions should never be declared in javadocs. That is just odd to me.